File: conformance/bundles/17-unexpected-pass/quiet.steps.py
from varar import steps

# The example carries an `error` fence, so it asserts a failure. This stimulus
# raises nothing, so the fence inverts into an UnexpectedPassError — the kind no
# bundle exercised before this one.
param, stimulus, sensor = steps()


@stimulus("I do nothing at all")
def _(state):
    return None

File: conformance/bundles/17-unexpected-pass/quiet.steps.cs
// C# sibling of quiet.steps.ts (bundle 17-unexpected-pass).
//
// The example carries an `error` fence, so it asserts a failure. This stimulus
// throws nothing, so the fence inverts into an UnexpectedPassError — the kind no
// bundle exercised before this one.
using Varar;
using Varar.Core;

namespace Varar.Corpus.B17;

public static class QuietSteps
{
    public static void Register(Steps s)
    {
        s.Stimulus("I do nothing at all", state => state);
    }

    public static Value State() => Value.Null;
}
